app/views/admin_general/_to_do_list.html.erb

```diff
@@ -26,7 +26,16 @@
                   <% end %>
                 <% elsif item.is_a? InfoRequest %>
                   <%= request_both_links(item) %>
-                  <% if InfoRequest.requires_admin_states.include?(item.described_state) %>
+                  <% if item.last_event.event_type == 'send_error' %>
```

Comment:
Put `send_error` in `InfoRequest.requires_admin_states` and then tweak the conditional on L50

Reply:
ah, the `described_state` here is `error_message` which is already there - just need to improve the conditional